src/etc/di.xml

```diff
@@ -30,6 +30,27 @@
             <argument name="instanceName" xsi:type="string">Magento\Catalog\Model\ResourceModel\Product\Attribute\Collection</argument>
         </arguments>
     </virtualType>
+    <virtualType name="Omikron\Factfinder\Model\Export\ExportPreviewProductFeed" type="Omikron\Factfinder\Model\Export\Feed">
```

Comment:
can we avoid defining the columns again? Can a virtual type extend already existing type?